src/azureFunction/azureFunctionUtils.ts

```diff
@@ -0,0 +1,90 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import * as fse from 'fs-extra';
```

Comment:
Same comment about the other dependency. Do we really need fs-extra, or can we use fs?

Reply:
Unable to use fs as it uses writeFileSync which would need the contents to be JSON, and since we are putting a map of contents instead fse allows us to do that instead directly.

Reply:
I'm not really sure what you mean by "putting a map of contents" @VasuBhog , but unless I'm missing something the reason is because we're writing the JSON and want to pretty-print it. But `JSON.stringify` takes a parameter for that so is there any reason we could just use fs.promises.writeFile(...) with the JSON.stringify content of settings?

(I realize that this is just copied code from AF but I agree with Chris that we should avoid extra dependencies unless we actually need them)